datasets.py

```diff
@@ -35,40 +35,31 @@ class Dataset:
 
     @classmethod
     def from_text(cls, text: str) -> Self:
-        """Convert a text representing a single dataset into the `Dataset` instance.
+        """Convert a text representing a single dataset into the `Dataset` instance."""
 
-        Args:
-            text (str): String with specific pattern in which each row is separated
-                by "\n"
-
-        Raises:
-            Exception: Input text does not have enough rows to assign values to
-                all attributes of the Dataset class.
-
-        Returns:
-            Self: Returns Dataset class object with data imported from input text.
-        """
-        if text.find("*") != -1:
+        typesL = Literal["density", "viscosity", "surften"]
```

Comment:
A minor comment ref. naming (`typesL` smells like JS...). I would go with `TDatasetType` and move this definition to the top of the module to avoid repetition (DRY rule).